pkg/reconciler/artifactbuild/artifactbuild.go

```diff
@@ -567,6 +573,40 @@ func (r *ReconcileArtifactBuild) handleCommunityDependencies(ctx context.Context
 	return nil
 }
 
+func (r *ReconcileArtifactBuild) updateLabel(ctx context.Context, log logr.Logger, abr *v1alpha1.ArtifactBuild) (bool, error) {
+	result := false
+
+	if abr.Labels == nil {
+		abr.Labels = make(map[string]string)
```

Comment:
It does not really matter, but everywhere else in the code I have been using `map[string]string{}` to create an empty map.

Reply:
When I looked in the golang docs on how to create maps, most seemed to highlight the make built-in which is why I used that I'm not sure if there is any difference internally between the two? From what I can see there doesn't seem to be though the make allows 'hinting' of initial allocation requirements.
I've done a search and while the code is using both patterns, Gabe used both on the same commit :-)